docs/source/deployment/deploy_sre.md

```diff
@@ -164,6 +176,28 @@ As some general recommendations,
 
 :::
 
+### Copy and paste
+
+The [Guacamole clipboard](https://guacamole.apache.org/doc/gug/using-guacamole.html#using-the-clipboard) provides an interface between the local clipboard and the clipboard on the remote workspaces.
+Only text is allowed to be passed through the Guacamole clipboard.
+
+Copy and pasting of text to or from SRE workspaces via the Guacamole clipboard can be configured with the `allow_copy` and `allow_paste` settings.
+
+`allow_copy` allows users to copy text from an SRE workspace to the Guacamole clipboard.
+
+`allow_paste` allows users to paste text into an SRE workspace from the Guacamole clipboard.
+
+These options have no impact on the ability to use copy and paste within a workspace.
+
+The impact of setting each of these options is detailed in the below table.
+
+| allow_copy | allow_paste | Copy/paste within workspace | Copy/paste between workspaces | Copy to local machine | Paste from local machine |
+|-----|-----|-----|-----|-----|-----|
+| true | true | yes | yes | yes | yes |
+| true | false |  yes | no | yes | no |
+| false | true | yes | no | no | yes|
+| false | false | yes | no | no | no |
```

Comment:
We should wrap this in a [table directive](https://mystmd.org/guide/tables) so we can add captions and cross reference.

It would also be good to have column spans for the inputs (`allow_copy`, `allow_paste`) and the outputs.

Reply:
Not sure if column spans are possible in (this flavour of) markdown?

Reply:
No, but we can do it with HTML (Markdown tables are annoying anyway 😆). I do think it will make it clearer what we are trying to convey.

Reply:
Updated with a html table, so ready to look at again